feedback-examples/streamlit/main.py

```diff
@@ -74,7 +81,6 @@ def _get_openai_type(msg):
         return msg.role
     return msg.type
```

Comment:
Please don't delete the comment unless you want to update the app to 1.26 and remove the whole logic :)